src/main.cpp

```diff
@@ -35,15 +36,18 @@ int main(int argc, char *argv[])
   bool fitAspectAxisToWindow = false;
   std::string valid_aspects = "alp"; // all, landscape, portait
   std::string overlay = "";
+  QString overlayHexRGB = QString();
+  QRegularExpression hexRGBMatcher("^#([0-9A-F]{3}){1,2}$", QRegularExpression::CaseInsensitiveOption);
```

Comment:
does it need to accept upper-case only?
According to [this example](https://cpp.hotexamples.com/examples/-/QColor/setNamedColor/cpp-qcolor-setnamedcolor-method-examples.html) it doesn't, but I haven't tested on my own.

Reply:
It does not, and I think `QRegularExpression::CaseInsensitiveOption` would let me match both cases.
I could update the regular expression to explicitly match both upper and lower cases.